generators/app/templates/src/__tools-jest._jest.config.js

```diff
@@ -0,0 +1,17 @@
+// https://github.com/thymikee/jest-preset-angular#brief-explanation-of-config
+module.exports = {
+  preset: 'jest-preset-angular',
+  roots: ['src'],
+  coverageDirectory: 'reports',
+  setupFilesAfterEnv: ['<rootDir>/src/setup-jest.ts'],
+  moduleNameMapper: {
+    '@app/(.*)': '<rootDir>/src/app/$1',
+    '@env': '<rootDir>/src/environments/environment'
+  },
+  // Do not ignore librairies such as ionic, ionic-native or bootstrap to transform them during unit testing.
+<% const excludedLibrairies = ['jest-test']
+  if (props.target.includes('cordova')) { excludedLibrairies.push('@ionic-native'); }
+  if (props.ui === 'ionic') { excludedLibrairies.push('@ionic'); }
+  if (props.ui === 'bootstrap') { excludedLibrairies.push('@ng-bootstrap'); } -%>
+  transformIgnorePatterns: ['node_modules/(?!(<%- excludedLibrairies.join('|') %>))']
+};
```

Comment:
If I remember well, the Jest config can also go to `package.json`, under the `jest` key.

As we already have (too) many config file, I'm all in for this move (can be done in another PR), any opinions on this? (@bursauxa ?)

Reply:
You remember well. I would go for the solution recommended by the Jest docs, as it turns out they do not recommend any of the two options in particular but package.json is mentioned first and jest.config.ts second:

https://jestjs.io/docs/en/configuration

So in that case I think package.json is the preferred option.